packages/a11y-base/src/focus-mixin.js

```diff
@@ -31,7 +31,7 @@ export const FocusMixin = dedupingMixin(
         });
 
         this.addEventListener('focusout', (e) => {
-          if (this._shouldRemoveFocus(e)) {
+          if (this.hasAttribute('focused') && this._shouldRemoveFocus(e)) {
```

Comment:
I don't quite understand how it can be so that field does NOT have focus, but has `focused` attribute. Please add a comment clarifying it, preferably with the link to web component reproduction.

Reply:
UPD: now I actually understand it. The actual reason of the root issue is having two focusables.

Reply:
Added a code comment.